Source/bmalloc/bmalloc/Environment.cpp

```diff
@@ -83,6 +90,12 @@ static bool isMallocEnvironmentVariableImplyingSystemMallocSet()
             return true;
     }
 
+    // Use system malloc anytime MallocStackLogging is enabled, except when the "vm" or "vmlite" logging modes are enabled.
+    // Those modes only intercept syscalls rather than mallocs, so they don't necessarily imply the use of system malloc.
+    const char* mallocStackLogging = getenv("MallocStackLogging");
+    if (mallocStackLogging && strncmp(mallocStackLogging, "vm", 2))
```

Comment:
I think this is not working only when `lite` is specified. So we should allow the other options.

Reply:
But the problem is only for `lite`, right? I can see correct memgraph if `MallocStackLogging=full`.

Reply:
Ok, I am not sure if memgraphs in WebContent are correct when full MSL is enabled since testing that that requires changing the WebContent sandbox. But restricting this to just lite seems fine.